common/biovotion/biovotion_vsm_blood_pulse_wave.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.biovotion",
+  "type": "record",
+  "name": "BiovotionVSMBloodPulseWave",
+  "doc": "Blood pulse wave data calculated by biovotion device.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "BloodPulseWaveValue", "type": "float", "doc": "Blood pulse wave value"},
```

Comment:
would it be ok to use lower camel case for field names? What kind of values can we expect here? Does it have a unit?